Dockerfile.rhel

```diff
@@ -0,0 +1,14 @@
+# This Dockerfile builds an image containing the Linux, Mac and Windows version of odo
+# layered on top of the ubi7/ubi image.
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
+
+COPY . /go/src/github.com/openshift/odo
+WORKDIR /go/src/github.com/openshift/odo
+RUN make cross
+
+FROM registry.access.redhat.com/ubi7/ubi
+COPY --from=builder /go/src/github.com/openshift/odo/dist/bin/darwin-amd64/odo /usr/share/openshift/odo/mac/odo
+COPY --from=builder /go/src/github.com/openshift/odo/dist/bin/windows-amd64/odo.exe /usr/share/openshift/odo/windows/odo.exe
+COPY --from=builder /go/src/github.com/openshift/odo/dist/bin/linux-amd64/odo /usr/share/openshift/odo/linux/odo
+LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts
```

Comment:
This line should be moved to right after `FROM`, It will improve caching and speed up build a tiny bit ;-)

Reply:
hmm, make sense